assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -39,22 +34,33 @@ import { __ } from '@wordpress/i18n';
 import { Section } from '../../../common';
 import PillGroup from '../../shared/pillGroup';
 import { PANE_PADDING } from '../../shared';
-import useRovingTabIndex from '../../../../../utils/useRovingTabIndex';
 import localStore, {
   LOCAL_STORAGE_PREFIX,
 } from '../../../../../utils/localStore';
 import { UnitsProvider } from '../../../../../units';
 import { PAGE_RATIO, TEXT_SET_SIZE } from '../../../../../constants';
 import useLibrary from '../../../useLibrary';
 import TextSet from './textSet';
-import { TEXT_SET_ACTIONS, getTextSetReducer } from './getTextSetReducer';
+
+const TEXT_SET_ROW_GAP = 12;
 
 const TextSetContainer = styled.div`
+  height: ${({ height }) => `${height}px`};
+  width: 100%;
+  position: relative;
+  margin-top: 28px;
```

Comment:
```suggestion
  margin-top: 28px;
  overflow-x: hidden;
```

adding overflow-x hidden here will prevent this excess scroll in firefox that's happening here because positioning is getting read a little different from chrome. 

![firefox horizontal scroll](https://user-images.githubusercontent.com/10720454/96161740-8278bc80-0ecc-11eb-955f-7928616efa46.gif)